x-pack/test/apm_api_integration/tests/alerts/error_count_threshold.spec.ts

```diff
@@ -69,7 +69,8 @@ export default function ApiTest({ getService }: FtrProviderContext) {
       await synthtraceEsClient.clean();
       await supertest.delete(`/api/alerting/rule/${ruleId}`).set('kbn-xsrf', 'foo');
       await supertest.delete(`/api/actions/connector/${actionId}`).set('kbn-xsrf', 'foo');
-      await esDeleteAllIndices(['.alerts*', INDEX_NAME]);
+      await esDeleteAllIndices(INDEX_NAME);
+      await es.deleteByQuery({ index: '.alerts*', query: { match_all: {} } });
```

Comment:
Because the indices are installed on Kibana start and not on first rule execution, we don't want to delete the index. Instead we delete the alert docs in the index so the tests have a fresh index.